quickwit/quickwit-indexing/src/actors/indexing_service.rs

```diff
@@ -1605,7 +1605,7 @@ mod tests {
         let observation = indexing_server_handle.process_pending_and_observe().await;
         assert_eq!(observation.num_running_pipelines, 0);
         assert_eq!(observation.num_running_merge_pipelines, 0);
-        universe.sleep(*HEARTBEAT).await;
+        universe.sleep(2 * *HEARTBEAT).await;
```

Comment:
ah ok I think I understand why, the supervise loop [actually runs every 1s ](https://github.com/quickwit-oss/quickwit/blob/f843b8858a34de5d6941a476d30bc95828f51e29/quickwit/quickwit-indexing/src/actors/merge_pipeline.rs#L483)

It happens to be that `2 * *HEARTBEAT` is also 1s, but I think it would be more explicit to create a `SUPERVISE_LOOP_INTERVAL` const and use it here and in the `SuperviseLoop` handler.